react-nextjs/react-host-remote/e2e/CommonCheck.cy.ts

```diff
@@ -0,0 +1,73 @@
+import {BaseMethods} from "../../../cypress/common/base";
+import {baseSelectors} from "../../../cypress/common/selectors";
+import {Constants} from "../../../cypress/fixtures/constants";
+const basePage: BaseMethods = new BaseMethods()
+
+const appsData = [
+    {
+        appName: Constants.elementsText.reactHostRemoteContainerHeader,
+        appDiv: Constants.elementsText.reactHostRemoteContainerDiv,
+        host: 8080
+    },
+
+    {
+        appName: Constants.elementsText.reactHostRemoteHostedDiv,
+        appDiv: Constants.elementsText.reactHostRemoteContainerDiv,
+        appButtonInvoices: Constants.elementsText.reactHostRemoteHostedInvoices,
+        appButtonExpenses: Constants.elementsText.reactHostRemoteHostedExpenses,
+        appButtonInvoicesH2: Constants.elementsText.reactHostRemoteHostedInvoices,
+        appButtonExpensesH2: Constants.elementsText.reactHostRemoteHostedExpenses,
+        host: 8081
+
+    }
+]
+
+appsData.forEach((
+        property: {
+            appName: string,
+            appDiv: string,
+            host: number,
+            appButtonInvoices?: string,
+            appButtonExpenses?: string,
+            appButtonInvoicesH2?: string,
+            appButtonExpensesH2?: string
+        }
+    ) => {
+describe(`Check react-host-remote ${property.appName} starts and running`, () => {
+    before(() => {
+        basePage.openLocalhost(property.host)
+    })
+    it (`Check Host ${property.appName} UI and Invoices, Expenses buttons`, () => {
+        basePage.checkElementWithTextPresence({
+            selector: baseSelectors.divElement,
+            text: property.appName
+        })
+        basePage.checkElementWithTextPresence({
+            selector: baseSelectors.divElement,
+            text: property.appDiv
+        })
+        if(property.host==8081){
+            basePage.clickElementWithText({
+                selector: baseSelectors.linkTag,
+                text: property.appButtonInvoices
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h2,
+                text: property.appButtonInvoicesH2
+            })
+            basePage.goBack()
+            basePage.clickElementWithText({
+                selector: baseSelectors.linkTag,
+                text: property.appButtonExpenses
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h2,
+                text: property.appButtonExpensesH2
+            })
+        }
+
+    })
+})
+    }
```

Comment:
some strange bracket position. I suppose you have some bracket error somewhere

Reply:
used reformatCode, command+option+l
please check